pkg/terraform/terraform.go

```diff
@@ -127,3 +134,30 @@ func unpackAndInit(dir string, platform string) (err error) {
 	}
 	return nil
 }
+
+func setupEmbeddedPlugins(dir string) error {
+	execPath, err := os.Executable()
+	if err != nil {
+		return errors.Wrapf(err, "failed to find path for binary %q", "openshift-install")
+	}
+
+	pdir := filepath.Join(dir, "plugins")
+	if err := os.MkdirAll(pdir, 0777); err != nil {
+		return err
+	}
+	for name := range plugins.KnownPlugins {
+		dst := filepath.Join(pdir, name)
+		if runtime.GOOS == "windows" {
+			dst = fmt.Sprintf("%s.exe", dst)
+		}
+		if _, err := os.Stat(dst); err == nil {
+			// stat succeeded, the plugin already exists.
```

Comment:
Does this happen?  I'd rather fix our code to not call this twice on the same directory.